src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/GroupModelRunParameterControllerTests.kt

```diff
@@ -31,12 +33,29 @@ class GroupModelRunParameterControllerTests : MontaguTests()
             on { params(":touchstone-id") } doReturn "touchstone-1"
             on { params(":scenario-id") } doReturn "scenario-1"
         }
-
         val repo = mockRepository(modelRunParameterSets = modelRunParameterSets)
-        val controller = GroupModelRunParametersController(mockContext, repo)
+
+        val controller = GroupModelRunParametersController(mockContext, repo, mock())
         assertThat(controller.getModelRunParameterSets()).isEqualTo(modelRunParameterSets)
     }
 
+    @Test
+    fun `can get model run params csv`()
+    {
+        val context = mock<ActionContext> {
+            on { it.params(":group-id") } doReturn "gId"
+            on { it.params(":touchstone-id") } doReturn "touchstone-1"
+            on { it.params(":model-run-parameter-set-id") } doReturn "1"
+            on { hasPermission(any()) } doReturn true
+        }
+
+        val data = GroupModelRunParametersController(context, mockRepository(), mockModellingGroupRepository())
+                .getModelRunParameterSet()
+
+        assertThat(data.contentType).isEqualTo("text/csv")
+        assertThat(data.data.first().runId).isEqualTo("1")
+    }
+
```

Comment:
Would be helpful to test that this method will throw an error if the touchstone isn't open and the user doesn't have permissions to view in preparation touchstones